.github/workflows/ci.yml

```diff
@@ -106,11 +113,26 @@ jobs:
             dylib: post.dll
             staticlib: post.lib
     steps:
+      - if: matrix.os == 'ubuntu-20.04'
+        run: sudo apt-get install -y libpocl2 mesa-opencl-icd ocl-icd-opencl-dev
+        name: Install opencl
+      - if: matrix.os == 'windows-latest'
+        name: Install opencl
+        run: vcpkg install opencl
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
         with:
           key: ${{ join( matrix.os, '-' ) }}
+
+      - if: matrix.os == 'windows-latest'
+        name: Download OpenCL.lib
+        run: |
+          pwd
+          choco install -y wget unzip
+          wget https://github.com/KhronosGroup/OpenCL-SDK/releases/download/v2023.04.17/OpenCL-SDK-v2023.04.17-Win-x64.zip
+          unzip -j OpenCL-SDK-v2023.04.17-Win-x64.zip OpenCL-SDK-v2023.04.17-Win-x64/lib/OpenCL.lib
```

Comment:
Can this file be added to the windows `.zip` file during the build? Otherwise we would need to document which version of the library we used fro which version of `post-rs`.

Reply:
I'm not sure why. OpenCL.lib is linked statically. The same happens for Mac and Linux (except the lib is fetched differently).

Perhaps we should include this information in release notes.